tests/unit/test_indicator.py

```diff
@@ -0,0 +1,415 @@
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_relative_strength_index(self):
+        """ test the relative_strength_index """
+        r_cudf = relative_strength_index(self._cudf_data['High'],
+                                         self._cudf_data['Low'], 10)
+        r_pandas = p.relative_strength_index(self._pandas_data, 10)
+        err = error_function(r_cudf, r_pandas['RSI_10'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+    @ordered
+    def test_mass_index(self):
+        """ test the mass_index """
+        r_cudf = mass_index(self._cudf_data['High'],
+                            self._cudf_data['Low'], 9, 25)
+        r_pandas = p.mass_index(self._pandas_data)
+        err = error_function(r_cudf, r_pandas['Mass Index'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+    @ordered
+    def test_true_strength_index(self):
+        """ test the true_strength_index """
+        r_cudf = true_strength_index(self._cudf_data['Close'], 5, 8)
+        r_pandas = p.true_strength_index(self._pandas_data, 5, 8)
+        err = error_function(r_cudf, r_pandas['TSI_5_8'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_chaikin_oscillator(self):
+        """ test the chaikin_oscillator """
+        r_cudf = chaikin_oscillator(self._cudf_data['High'],
+                                    self._cudf_data['Low'],
+                                    self._cudf_data['Close'],
+                                    self._cudf_data['Volume'],  3, 10)
+        r_pandas = p.chaikin_oscillator(self._pandas_data)
+        err = error_function(r_cudf, r_pandas['Chaikin'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_money_flow_index(self):
+        """ test the money_flow_index """
+        r_cudf = money_flow_index(self._cudf_data['High'],
+                                  self._cudf_data['Low'],
+                                  self._cudf_data['Close'],
+                                  self._cudf_data['Volume'], 10)
+        r_pandas = p.money_flow_index(self._pandas_data, 10)
+        err = error_function(r_cudf, r_pandas['MFI_10'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_on_balance_volume(self):
+        """ test the on_balance_volume """
+        r_cudf = on_balance_volume(self._cudf_data['Close'],
+                                   self._cudf_data['Volume'], 10)
+        r_pandas = p.on_balance_volume(self._pandas_data, 10)
+        err = error_function(r_cudf, r_pandas['OBV_10'])
+        msg = "bad error %f\n" % (err,)
+        self.assertTrue(np.isclose(err, 0, atol=1e-6), msg)
+
+    @ordered
+    def test_money_flow_index(self):
```

Comment:
Duplicated signature for the test. You also have a test named `test_money_flow_index` on line 275 above. Rename one of the tests.